src/blackdoor/cqbe/addressing/AddressTable.java

```diff
@@ -26,7 +27,16 @@
  * @version v1.0.0 - Nov 19, 2014
  */
 public class AddressTable extends ConcurrentSkipListMap<byte[], L3Address> implements Serializable, Iterable<L3Address> {
-	public static final int DEFAULT_MAX_SIZE = (int) Config.getReadOnly("default.config", "address_table_max_size");
+	public static final int DEFAULT_MAX_SIZE;
+	static{
+		int temp;
+		try{
+			temp = (int) Config.getReadOnly("default.config", "address_table_max_size");
+		}catch(ConfigFileNotFoundException e){
+			temp = 128;
+		}
+		DEFAULT_MAX_SIZE = temp;
+	}
```

Comment:
when i did the fix for this I didn't notice that the new method swiched the order of the files to  Config.getReadOnly("address_table_max_size", "default.config")
if you look at my pull reset I fixed it there no need for this try catch since the default.config will always be there